images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -41,6 +41,7 @@ if [[ -f ${APP_ROOT_PERSISTENT_VMDB}/config/database.yml && -f ${PV_DEPLOY_INFO_
   echo "== Found existing deployment configuration =="
   echo "== Restoring existing database configuration =="
   ln --backup -sn ${APP_ROOT_PERSISTENT_VMDB}/config/database.yml ${APP_ROOT}/config/database.yml
+  ln --backup -sn ${APP_ROOT_PERSISTENT_VMDB}/certs/v2_key ${APP_ROOT}/certs/v2_key
```

Comment:
Yes I can add a check for v2_key existance for safety, will update. No need to cleanup keys as we should not be creating or copying DB keys during build-time to begin with, ever.